code/math/implementation.lisp

```diff
@@ -38,15 +189,101 @@
          (ash p ,(- size)))))
 
 (defun round-to-odd/32 (g cp)
```

Comment:
Let's put these in the same order as floor-multiply (swap g and cp) and use the same naming scheme.